export/examples/http/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"time"
+
+	"github.com/confluentinc/telemetry-reporter-go/export"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+)
+
+var (
+	metric1 = stats.Float64("metric1", "first dummy metric for test script for HTTP exporter", "ms")
+	metric2 = stats.Int64("metric2", "second dummy metric for test script for HTTP exporter", "By")
+)
+
+// var (
+// 	// KeyMethod ...
+// 	KeyMethod, _ = tag.NewKey("method")
+
+// 	// KeyStatus ...
+// 	KeyStatus, _ = tag.NewKey("status")
+
+// 	// KeyError ...
+// 	KeyError, _ = tag.NewKey("error")
+// )
```

Comment:
Do you want to delete this commented out code? Or if it's here for an example, could you put a descriptive comment at the top of the comment block? Maybe we'd add it to the top-level package comment in that case?